mnefun/_mnefun.py

```diff
@@ -1888,7 +1913,7 @@ def do_preprocessing_combined(p, subjects, run_indices):
 
         proj_nums = p.proj_nums
         eog_t_lims = [-0.25, 0.25]
-        ecg_t_lims = [-0.08, 0.08]
+        ecg_t_lims = [-0.2, 0.2]
```

Comment:
why this change? It should probably be a parameter if it needs tweaking, as this is a backward-incompatible change

Reply:
I was just playing around with this and forgot to change it back. I was curious as to why you went with such a narrow window for QRS epoching? In the mne tutorials, the window is -500 - 500 ms and I've been using -200 - 200 ms for infants. Why is this backward-incompatible?

Reply:
>  was curious as to why you went with such a narrow window for QRS epoching?

I don't remember but I doubt it makes a huge difference

> Why is this backward-incompatible?

It will change the computations and thus the results you get if you run `mnefun` before / after this PR (and not in a bug-fixing sort of way)